dolphinscheduler-task-plugin/dolphinscheduler-task-datax/src/main/java/org/apache/dolphinscheduler/plugin/task/datax/DataxUtils.java

```diff
@@ -88,6 +94,8 @@ public static String getWriterPluginName(DbType dbType) {
                 return DATAX_WRITER_PLUGIN_CLICKHOUSE;
             case DATABEND:
                 return DATAX_WRITER_PLUGIN_DATABEND;
+            case XUGU:
```

Comment:
<img width="1438" alt="image" src="https://github.com/apache/dolphinscheduler/assets/33984497/97548610-e2fc-46ff-9a0f-5555aa3e5ba3">
hi @mrliufox   I couldn't find the Xugu reader and writer on the official DataX website. I suggest removing this section and hiding the Xugu data source in the frontend module that relation to DATAX  cc @zhuangchong @ruanwenjun